src/modules/crud.js

```diff
@@ -0,0 +1,135 @@
+import deleteCompletedTasks from './deleteCompletedTasks.js';
+
+// Get HTML elements
+const addButton = document.getElementById('add-button');
+const input = document.getElementById('add-item');
+
+// Tasks on the local storage
+let ItemsArray = JSON.parse(localStorage.getItem('tasks')) || [];
+
+function displayTasks() {
+  // Get the ul element
+  const list = document.querySelector('#list');
+  list.innerHTML = '';
+
+  // Loop through the items and add them to the ul
+  ItemsArray.forEach((task) => {
+    // Create the li element
+    const li = document.createElement('li');
+    li.classList.add('item');
+
+    // Create the <input> element
+    const checkbox = document.createElement('input');
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.setAttribute('name', 'item');
+    checkbox.setAttribute('id', `item-${task.index}`);
+    console.log(task.index);
+    checkbox.setAttribute('data-index', task.index);
+    checkbox.checked = task.completed;
+    li.appendChild(checkbox);
+
+    // Create the <label> element
+    const label = document.createElement('label');
+    label.setAttribute('for', `item-${task.index}`);
+    label.textContent = task.description;
+    li.appendChild(label);
+
+    // Create the edit button
+    const editButton = document.createElement('button');
+    editButton.classList.add('btn');
+    li.appendChild(editButton);
+
+    // Create the delete button
+    const deleteButton = document.createElement('button');
+    deleteButton.classList.add('btn-delete');
+    li.appendChild(deleteButton);
```

Comment:
The code for creating edit and delete buttons don't show for me in my live server so I was wondering if there's something missing so the user can interact with them. 😅 
![Ruthcode](https://user-images.githubusercontent.com/120128067/232667511-47300b39-c40c-4ace-bde5-e3d8c3ce8514.png)

Reply:
Hahahah My bad, I forgot to upload the images. And validated that about the empty task.
Thank you very much, Melanie 😄